### ISSUE TITLE
create dirs for --output

### ISSUE THREAD

**Issue by KernelDev** [Filiosoft/npm-audit-html#6]
Hello,

as I was using the --output option, it did not create missing directories in the given path. This behavior is expected by most of the users, til they are used to mocha reporters or nyc reporters.

Command invoked:
``` sh
npm audit --json | npm-audit-html --output ./test-reports/npm-audit/report.html
``` 
the folder npm-audit does not exist and should be generated by the module

**Comment by nprail** [comment]
@KernelDev After looking at it again, I'm not sure that this is a use case that I want to support. For now, I have added an error message saying that it isn't supported.

**Comment by ghost** [comment]
> 🎉 This issue has been resolved in version 1.2.0 🎉
> 
> The release is available on:
> 
> * [npm package (@latest dist-tag)](https://www.npmjs.com/package/npm-audit-html)
> * [GitHub release](https://github.com/Filiosoft/npm-audit-html/releases/tag/v1.2.0)
> 
> Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot 📦🚀

Sorry for missing this so long. Thank you very much for the fix. Works like a charm. Sorry for my late response.

**Comment by nprail** [comment]
@KernelDev Awesome, glad to hear that!
